### DISCUSSION TITLE
where to download the old version of Eclipse plugins which DBeaver

### DISCUSSION THREAD

**Discussion by a2bothic** [dbeaver/dbeaver#21133] [DBeaver Development] [answered]
offline develop with version dbeaver-23.1.4
Need plugins bcprov_1.72.0.jar, not find in https://p2.dev.dbeaver.com/eclipse-repo/plugins/.
Where can I download it

**Comment by E1izabeth** [comment, marked as answer]
You could try to get it with Install New Software dialog and this repo https://download.eclipse.org/tools/orbit/downloads/drops/R20221123021534/repository/ (this url is written in pom in <orbit-repo.url> tag)

**Comment by shadrukhin** [comment]
Hi, but what about getting DBeaver Eclipse Plugin at all?
I need it for version 24.1.2 
Can i get it anywhere?
